integration/openshift/setup.sh

```diff
@@ -51,11 +51,13 @@ popd
 
 echo "Install Openshift"
 openshift_repo="github.com/openshift/origin"
-go get -d "$openshift_repo" || true
-pushd "$GOPATH/src/$openshift_repo"
-git checkout "$origin_version"
-export OS_OUTPUT_GOPATH=1
-make
-sudo cp _output/local/bin/linux/amd64/{openshift,oc,oadm} /usr/bin
-popd
+openshift_tarball="openshift-origin-server-${origin_version}-c4dd4cf-linux-64bit.tar.gz"
```

Comment:
I think the `c4dd4cf` is part of, or at least tied to, the `origin_version` - so probably needs to be a var or part of that var

Reply:
Right, I added a new var called `origin_commit`